freqtrade/exchange/binance.py

```diff
@@ -227,3 +228,11 @@ async def _async_get_historic_ohlcv(self, pair: str, timeframe: str,
                             f"{arrow.get(since_ms // 1000).isoformat()}.")
         return await super()._async_get_historic_ohlcv(
             pair=pair, timeframe=timeframe, since_ms=since_ms, is_new_pair=is_new_pair)
+
+    def funding_fee_cutoff(self, d: datetime):
+        '''
+            # TODO-lev: Double check that gateio, ftx, and kraken don't also have this
+            :param d: The open date for a trade
+            :return: The cutoff open time for when a funding fee is charged
+        '''
+        return d.minute > 0 or (d.minute == 0 and d.second > 15)
```

Comment:
Do i understand this correctly - that binance still applies the fee if the trade is opened within the first 15 seconds of the "funding" hour? 
won't we need to be careful here as it'll not apply to 13:00, but only to 12:00, 16:00, ... as binance only applies fees every 4 hours? (don't consider it a change request if i'm wrong :laughing: )

Reply:
Ok, I added a 1 am test, and yeah that's correct. I emailed the other 3 exchanges seeing if they had something similar

Reply:
FTX, said no

Kraken said

"""
So  if a trader opened a position at 08:00:05 UTC, and closed it at  08:30:00 UTC, would this trader be charged the 08:00:00 UTC funding fee? Please note that the absolute funding payout can be calculated by using the formula below:Absolute funding payout = # of contracts * absolute funding rate * time elapsed within funding period without position alterationAs  you can see above, the funding payout is calculated based on the time  your position remains open and unaltered within the funding period. In  your example, this would be 29 min and 55 seconds.
-- | --
"""

Reply:
I just double checked with binance

```
If two traders are holding a long position for 300 ADA on the ADA/USDT perpetual futures market, but one entered their position at 02:00 UTC on Nov 1 2021, and the other entered their position at 06:00 UTC on Nov 1 2021, then if they are both still holding their positions at 08:00 UTC on Nov 1 2021, they will be charged the same funding fee. Is that correct?
```

and they said it was

Reply:
so basically, on kraken, the "hour" is irrelevant, as it's the duration of the trade that counts ...